README.md

```diff
@@ -1,3 +1,7 @@
+conp8 : [![Binder](https://conp8.calculquebec.cloud/badge_logo.svg)](https://conp8.calculquebec.cloud/v2/gh/ltetrel/introML-book/master/?filepath=content/01)
```

Comment:
I don't think we want these links here -- we want folks to access the binder from the book.

Reply:
Ok I can remove it.
Why we want that ? I think it is nice to give the opportunity to access directly the binder, for advanced user who wants to bypass the book ?

Reply:
That should be a binder link on the original `introML` repo, rather than here :)

Reply:
Ok I get it ! Also we should think of the process of building the book automatically, we already discussed about it (but it is another discussion :D)